package.json

```diff
@@ -38,13 +38,13 @@
     "@kiosked/ulid": "^3.0.0",
     "@rollup/plugin-replace": "^2.3.3",
     "@rollup/plugin-strip": "^1.3.3",
-    "@wdio/browserstack-service": "5.13.2",
-    "@wdio/cli": "5.15.0",
-    "@wdio/junit-reporter": "5.15.0",
-    "@wdio/local-runner": "5.15.0",
-    "@wdio/mocha-framework": "5.15.0",
-    "@wdio/spec-reporter": "5.14.5",
-    "@wdio/sync": "5.14.4",
+    "@wdio/browserstack-service": "^6.12.0",
```

Comment:
version < 6.1.10 will not report status correctly to browserstack https://github.com/webdriverio/webdriverio/commit/85ee21d7aedb9ea1bcac0b754f5b19a799cbe1dc